modules/UI/videolayout/VideoContainer.js

```diff
@@ -576,6 +573,17 @@ export class VideoContainer extends LargeContainer {
     }
 
     /**
+     * Sets the blur background to be invisible and pauses any playing video.
+     *
+     * @private
+     * @returns {void}
+     */
+    _hideVideoBackground() {
+        this.$videoBackground.css({ visibility: 'hidden' });
```

Comment:
The change to hide was actually for temasys, which cannot interact with its video object if set to display: none.